.github/workflows/performance.yml

```
@@ -0,0 +1,143 @@
+
+      - name: Build and analyze Core Web Vitals
+        run: |
+          echo "🎯 Core Web Vitals Analysis"
+          echo "==========================="
+
+          # Build the application
+          npm run build
+
+          echo "📊 Core Web Vitals targets for excellent UX:"
+          echo "  • LCP (Largest Contentful Paint): ≤ 2.5s"
+          echo "  • FID (First Input Delay): ≤ 100ms" 
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)